# PopgenTools
A collection for available tools in population genetics. For more tools, see [here](https://omictools.com/population-genomics-category).

## Data Manipulation
- [BCFTOOLS](https://samtools.github.io/bcftools/bcftools.html)
- [VCFTOOLS](https://vcftools.github.io/index.html)
- [SAMTOOLS](http://www.htslib.org/)
- [TABIX](http://www.htslib.org/doc/tabix.html)
- [GATK](https://software.broadinstitute.org/gatk/)
- [GTOOL](http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html)
- [htslib](https://github.com/samtools/htslib)
- [htsjdk](https://github.com/samtools/htsjdk)
- [Picard](https://broadinstitute.github.io/picard/)
- [PLINK 1.7](http://zzz.bwh.harvard.edu/plink/tutorial.shtml)
- [PLINK 1.9](https://www.cog-genomics.org/plink2)
- [scikit-allel](https://scikit-allel.readthedocs.io/en/latest/)

## Genotype Phasing and Imputation
- [BEAGLE](https://faculty.washington.edu/browning/beagle/beagle.html)
- [IMPUTE](https://mathgen.stats.ox.ac.uk/impute/impute_v1.html)
- [IMPUTE2](http://mathgen.stats.ox.ac.uk/impute/impute_v2.html)
- [Minimac3](https://github.com/Santy-8128/Minimac3)
- [SHAPEIT](https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html)
- [STITCH](http://www.stats.ox.ac.uk/~myers/)

## Natural Selection
- [4P](https://github.com/anbena/4p)
- [GCTA-MLMA](http://cnsgenomics.com/software/gcta/mlmassoc.html)
- [hapbin](https://github.com/evotools/hapbin)
- [hapFLK](https://forge-dga.jouy.inra.fr/projects/hapflk)
- [pcadapt](https://cran.r-project.org/web/packages/pcadapt/index.html)
- [REHH](https://cran.r-project.org/web/packages/rehh/index.html)
- [SCCT](https://github.com/wavefancy/scct)
- [SeleDiff](https://github.com/xin-huang/SeleDiff)
- [selscan](https://github.com/szpiech/selscan)
- [XPCLR](https://reich.hms.harvard.edu/software)

## Population Admixture and Structure
- [AdmixTools](https://github.com/DReichLab/AdmixTools)
- [ADMIXTURE](https://www.genetics.ucla.edu/software/admixture/)
- [EIGENSOFT](https://github.com/DReichLab/EIG)
- [fastStructure](https://rajanil.github.io/fastStructure/)
- [FlashPCA2](https://github.com/gabraham/flashpca)
- [MixMapper](https://reich.hms.harvard.edu/software)
- [SNPRelate](https://github.com/zhengxwen/SNPRelate)
- [STRUCTURE](http://web.stanford.edu/group/pritchardlab/structure.html)
- [TreeMix](https://bitbucket.org/nygcresearch/treemix/wiki/Home)

## Population Demography
- [ASMC](https://github.com/pierpal/ASMC)
- [continuity](https://github.com/schraiber/continuity)
- [dadi](https://bitbucket.org/gutenkunstlab/dadi)
- [moments](https://bitbucket.org/simongravel/moments)
- [MSMC](https://github.com/stschiff/msmc)
- [PSMC](https://github.com/lh3/psmc)
- [SMC++](https://github.com/popgenmethods/smcpp)

## Distribution of fitness effects
- [anavar](http://zeng-lab.group.shef.ac.uk/wordpress/)
- [dadi](https://bitbucket.org/gutenkunstlab/dadi)
- [DFE-alpha](http://www.homepages.ed.ac.uk/pkeightl/software.html)
- [polyDFE](https://github.com/paula-tataru/polyDFE)
- [prfreq](https://bustamantelab.stanford.edu/lab-developed-software)
- [predicDiv](https://github.com/LohmuellerLab/predicDiv)

## Genome-wide Association Study
- [GCTA](http://cnsgenomics.com/software/gcta/#Overview)
- [SKAT](https://www.hsph.harvard.edu/skat/)

## Genome annotation
- [ANNOVAR](https://doc-openbio.readthedocs.io/projects/annovar/en/latest/)
- [SnpEff](https://pcingola.github.io/SnpEff/)

## Simulation
- [cosi2](https://software.broadinstitute.org/mpg/cosi2/cosidoc.html)
- [ms](http://home.uchicago.edu/rhudson1/source/mksamples.html)
- [msms](http://www.mabs.at/ewing/msms/index.shtml)
- [Nemo](http://nemo2.sourceforge.net/)
- [SLiM](https://messerlab.org/slim/)

## Visualization
- [distruct](https://web.stanford.edu/group/rosenberglab/distruct.html)
- [Haploview](https://www.broadinstitute.org/haploview/haploview)
- [LocusZoom](http://locuszoom.sph.umich.edu/)
